Makefile

```diff
@@ -224,10 +230,12 @@ include $(MAKE_SCRIPT_DIR)/openocd.mk
 ifeq ($(CONFIG),)
 ifeq ($(TARGET),)
 .DEFAULT_GOAL := all
+else ifneq ($(filter $(TARGET),$(EXE_TARGETS)),)
+.DEFAULT_GOAL := exe
```

Comment:
Do not think we need exe target here ? Building SITL in master results in hex file